build.rs

```diff
@@ -10,13 +10,17 @@ fn main() {
     if cfg!(feature = "development") {
         return;
     }
-    if let Ok(target) = env::var("TARGET") {
-        if !target.starts_with("arm-") {
-            return;
-        }
-    }
 
     let out_dir = env::var("OUT_DIR").unwrap();
+
+    let target = env::var("TARGET").unwrap();
+    match target.as_str() {
+        "armv7-unknown-linux-gnueabihf" => {
+            env::set_var("CC", "arm-linux-gnueabihf-gcc");
```

Comment:
We have to be careful and avoid overwriting `CC` if it's already set from the outside. That ability was part of the latest release, so we shouldn't undo it. :wink: